src/fidocadj/circuit/controllers/HandleActions.java

```diff
@@ -9,6 +9,8 @@
 import fidocadj.primitives.PrimitiveRectangle;
 import fidocadj.primitives.PrimitiveMacro;
 import fidocadj.primitives.PrimitiveOval;
+import java.awt.Color;
```

Comment:
Do not use references to awt or Swing in such classes. The low-level graphic stuff should be accessed by means of GraphicInterface. This allowed to share the same code for the Android version. Even if I do not develop the Android version anymore, keeping things separated still is a good idea.